### ISSUE TITLE
Configuration platformaux seems to ignore fetchSources = false

### ISSUE THREAD

**Issue by Argelbargel** [stempler/bnd-platform#42]
If added a dependency to `platformaux``:
```
dependencies {
    platformaux 'the:dependency'
}
```
And disabled source-fetching in the platform-configuration:
```
platform {
    // ...
    fetchSources = false
}
```
While `fetchSources` seems to work for dependencies added to the `platform`-configuration, the source jars get fetched for those in `platformaux`.

**Comment by stempler** [comment]
Yes, seems [the sources are always added](https://github.com/stempler/bnd-platform/blob/83a5235bc8afa76d69c4e020cab68b9a2fd800de/src/main/groovy/org/standardout/gradle/plugin/platform/internal/BundlesAction.groovy#L135) in that case. Probably just needs a condition like [here](https://github.com/stempler/bnd-platform/blob/83a5235bc8afa76d69c4e020cab68b9a2fd800de/src/main/groovy/org/standardout/gradle/plugin/platform/internal/BundlesAction.groovy#L158) for that section.
